.JuliaFormatter.toml

```diff
@@ -0,0 +1 @@
+style = "blue"
```

Comment:
BlueStyle seems like the most well-established specification, whereas SciML is more recent. However JuliaFormatter cannot do it all, so we must tell contributors to adhere to the style guidelines in general. Should we add a badge to the README too?

Reply:
Any reason to prefer `BlueStyle`? We could also specify our own rules right?

Reply:
In any cases, most of the changes here seem to be sensible.

Reply:
I just use it because it's already well documented, so we can point contributors to the exhaustive specification by Invenia. We could make up our own rules but that would require explaining them to everyone in our docs.

Also, it's easier to contribute when you're already familiar with the style guide, which wouldn't be the case for Graphs-specific rules.

Reply:
My hot take is that the precise choice of style does not matter, as long as it's clear and easy to apply, and BlueStyle satisfies both criteria

Reply:
Maybe it makes sense to use some common style, as then people don't need to reconfigure their editor differently for every project.

Reply:
What about the default format by JuliaFormatter? Would that also be an option?

Reply:
The default format by JuliaFormatter is much less well-specified than BlueStyle. One of the perks of BlueStyle is that it also gives coherent writing advice to the contributors that cannot be enfored by a simple linter

Reply:
> Maybe it makes sense to use some common style, as then people don't need to reconfigure their editor differently for every project.

Another perk of JuliaFormatter is that the formatting in the Julia extension for VSCode is based on it. And it takes into account the `.JuliaFormatter.toml` file at the root of the package to format each file according to the local specification (in our case `style = "blue"`)